integrations/api_packages_nuget_test.go

```diff
@@ -24,9 +24,16 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func addNuGetAPIKeyHeader(request *http.Request, token string) *http.Request {
+	request.Header.Set("X-Nuget-Apikey", token)
```

Comment:
This would then be wrong as well...
```suggestion
	request.Header.Set("X-NuGet-ApiKey", token)
```